### PR TITLE
resolves 57

### LINKED ISSUE #57: iOS scrolling background of page instead of calendar; input focus problems;
### Description

On iPhone X, iOS 11.2, Safari, if I open the demo page, the following problems occur:

- When I click on any example (can be the first one), I see the calendar fullscreen; when I try to scroll upwards to see the next months (only january and february are within the viewport), I see the back of the page scolling instead, and only after trying 5 or 10 times I can manage to scroll (just a little) of the calendar that is fullscreen on the foreground;

- As soon as I click on any example, I see a very large blue cursor blinking inside the calendar, exactly where the <input> was. Any ideas how to remove this? It's very bothersome and kind of ruins the UI, has a very weird feel to it. 

- Also, even tough I can't reproduce enough, 30% of my clicks in the calendar end up selecting the wrong date, don't know exactly why. 

### Code sample 
Not needed, it's in the default examples. 

### Datepicker Version

None, it's in the examples.

### PR DESCRIPTION
**Pull request by chrisrenga**
resolves #57,
minor styling changes to fix ios overflow issue, replace date inputs with buttons to fix ios input selection issue on open datepicker, add a button to load the additional months on mobile

screen shot 2018-03-02 at 17 04 25

### CODE REVIEW

**Comment by krystalcampioni** [comment]
@chrisrenga thanks a lot for the contribution 😄
I checked out to your branch now and it's looking great apart from the situation described below:
When we try to scroll down to the next month on mobile, the arrow button also gets dragged to the top and we can see a white gap on the bottom of the page
screen shot 2018-04-28 at 8 20 51 am

Could you fix this issue so we can release your changes with the next version? 💪

**Comment by chrisrenga** [comment]
all done :)

File: src/components/DatePicker.vue [hunk 1]
```diff
@@ -94,6 +97,16 @@
...
+<<<<<<< HEAD
+            button.datepicker__month-button.datepicker__month-button--next.-hide-on-desktop(
+              @click='renderNextMonth' type="button"
+            )
+=======
+            div
+              button.next--mobile(
+                @click='renderNextMonth' type="button"
+              )
+>>>>>>> 64e79334f0de9cf2e5f7fa3ba06e73d417474740
```
[the remark this answers is not in the document]
Comment (chrisrenga) [reply src/components/DatePicker.vue]: ah, completely missed that one. will fix it now

**Comment by tbyte80** [comment]
Hello!
I found this PR because I having the same issue (on mobile it is a bit cumbersome to navigate through months. Often months wouldn't load especially on IOS). The button looks perfect to solve that but I cannot find how to activate it.
It doesn't appear on default and also in the examples I don't see it anywhere. 
Any advice?

**Pull request merged**